package.json

```diff
@@ -54,8 +54,8 @@
     "d3-format": "^1.2.0",
     "d3-time-format": "^2.0.5",
     "d3-selection": "^1.1.0",
-    "vega": "^3.0.0-beta.31",
-    "vega-lite": "^2.0.0-beta.3"
+    "vega": "^3.0.0",
+    "vega-lite": "^2.0.0-beta.14"
```

Comment:
I think these really should be peer dependencies. Let's fix that as soon as we have final releases.